### PR TITLE
[DA-129][DA-130] add methods for retrieving information from pom file

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
:-1:

**Comment by jbartece** [comment]
retest this please

File: communication/src/main/java/org/jboss/da/communication/scm/impl/SCMConnectorImpl.java [hunk 1]
```diff
@@ -53,4 +54,28 @@
...
+        if (!pomPath.endsWith("pom.xml")) {
+            pomPath += "/pom.xml";
+        }
+
+        try {
+            File tempDir = Files.createTempDirectory("cloned_repo").toFile();
+            try {
+                // git clone
+                // TODO: hardcoded to git right now
+                scmManager.cloneRepository(SCMType.GIT, scmUrl, revision, tempDir.toString());
```
Comment (jbartece) [inline communication/src/main/java/org/jboss/da/communication/scm/impl/SCMConnectorImpl.java]: We should extract this cloning of repositories to the extra method not to clone the same repository multiple times, but it is a problem in more beans so, we will do it at once. So, it is ok for this PR.

**Comment by jbartece** [comment]
:100: Nice use of JAXB

**Comment by jomrazek** [comment]
:+1:

**Pull request merged**